src/test/ui/const-generics/issue-99325.rs

```diff
@@ -0,0 +1,14 @@
+// build-pass
+// compile-flags: -Zunpretty=mir
```

Comment:
if you can reproduce with `-Zdump-mir=main` or `-Zdump-mir=function_with_bytes`, then you could turn this into a mir opt test instead of a ui test with mir output.